_posts/patterns/2015-03-25-headings.md

```diff
@@ -3,7 +3,7 @@ layout:         pattern
 title:          Headings
```

Comment:
I wonder if this would be a good opportunity to remove the dates from the files names? I think it confuses matters and doesn't really make sense to include them in this non-blog context.